CMakeLists.txt

```diff
@@ -32,7 +32,11 @@ IF(NOT KOKKOSKERNELS_HAS_TRILINOS)
     ENDIF()
   ENDIF()
   IF(NOT DEFINED ${PROJECT_NAME})
-    PROJECT(KokkosKernels CXX)
+    IF(${KokkosKernels_ENABLE_CUDA})
+      PROJECT(KokkosKernels CXX CUDA)
+    ELSE()
+      PROJECT(KokkosKernels CXX)
+    ENDIF()
```

Comment:
Wouldn't it be better to do something like
```cmake
if (NOT Kokkos_COMPILE_LANGUAGE STREQUAL CXX)
  PROJECT(KokkosKernels CXX ${Kokkos_COMPILE_LANGUAGE})
else()
  PROJECT(KokkosKernels CXX)
endif()
```
This approach would need more refactoring because I don't think `Kokkos` is found at that point.

Reply:
Ping